en/docs/develop/scim2-patch-operations.md

```diff
@@ -0,0 +1,928 @@
+            }
+        }
+    }
+    ```
+
+-   **Example 2:** Replace the `devices` attribue in a **custom** user schema (using the path param):
+
+    ```json
+    {
+        "op": "replace",
+        "path": "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User:devices",
+        "value": [
+            "M6",
+            "M7"
+        ]
+    }
+    ```
+
+**Complex multi-valued attributes**
+
+Consider the `emails` attribute in the **Core** user schema, which can have multiple emails types such as `home`, `work`, etc. Each email attribute has sub-attributes such as `value`, `type`, and `primary`.
+
+-   **Example 1:** Replace the `emails` attribute in a **custom** user schema (without using the path param):
+
+    ```json
+    {
+        "op": "replace",
+        "value": {
+            "emails": [
+                {
+                    "value": "abcd@gmail.com",
+                    "type": "home"
+                },
+                {
+                    "value": "wxyz@gmail.com",
+                    "type": "work"
+                }
+            ]
+        }
+    }
+    ```
+
+-   **Example 2:** Replace the `emails` attribute in a **custom** user schema (using the path param):
+
+    ```json
+    {
+        "op": "replace",
+        "path": "emails",
+        "value": [
+            {
+                "value": "abcde@gmail.com",
+                "type": "home"
+            },
+            {
+                "value": "vwxyz@gmail.com",
+                "type": "work"
+            }
+        ]
+    }
+    ```
+
+### Remove user attributes
+
+Let's create patch request payloads to `remove` user attributes of different types.
+
+**Simple singular attributes**
+
+Consider the `nickname` attribute in the **Core** user schema and the `country` attribute in the **Enterprise** user schema (`urn:ietf:params:scim:schemas:extension:enterprise:2.0:User`). These are simple singular attributes.
+
+-   **Example 1:** Remove the `nickName` attribute of a user in the **Core** user schema.
+
+    ```json
+    {
+        "op": "remove",
+        "path": "nickName"
+    }
+    ```
+
+-   **Example 2:** Remove the `country` attribute of a user in the **Enterprise** user schema.
+
+    ```json
+    {
+        "op": "remove",
+        "path": "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User:country"
+    }
+    ```
+
+**Complex singular attributes**
+
+Consider the `name` attribute in **Core** user schema, which has sub-attributes such as `givenName`, `familyName`, etc. and the `manager` attribute in the **Enterprise** user schema (`urn:ietf:params:scim:schemas:extension:enterprise:2.0:User`), which has `displayName`, `value`, `ref`, etc. as sub-attributes. 
+
+!!! Note    
+    -   If the path contains the complex attribute, all sub-attribute values are removed. 
+    -   There is a special case for the `name` attribute. Even though `“path”: “name”` is specified, the `familyName` attribute is not removed. 
```

Comment:
There is a special case for the `name` attribute. Even though `“path”: “name”` is specified, the `familyName` attribute is not removed. This special case is only for LDAP userstore. Let's mention that as well